pkg/client/request.go

```diff
@@ -556,7 +562,8 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) ([]b
 		// no-op, we've been upgraded
 	case resp.StatusCode < http.StatusOK || resp.StatusCode > http.StatusPartialContent:
 		if !isStatusResponse {
-			var err error = &UnexpectedStatusError{
+			var err error
+			err = &UnexpectedStatusError{
```

Comment:
More of a style thing, I think. The pattern is to either declare a variable with `var name type` and assign to it later or let the compiler infer type by assigning with `name := `. Doing both in the same statement is unnecessary.